### PR TITLE
removing new_http()

### PR DESCRIPTION
**Pull request by NickBenthem**

### CODE REVIEW

File: nflgame/update_players.py [hunk 1]
```diff
@@ -455,13 +455,24 @@ def fetch(purl):
...
-                if resp['status'] == '404':
-                    return gid, purl, False
+            resp = requests.get(purl)
+            if resp.status_code != 200:
+                if resp.status_code != 404:
+                    return gid,purl, False
                 else:
-                    return gid, purl, None
-            return gid, purl, content
+                    return gid, purl, nDone
```
Comment (ochawkeye) [inline nflgame/update_players.py]: What is `nDone`?

File: nflgame/version.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = '2.0.1b1'
+__version__ = '2.0.1a6'
```
Comment (ochawkeye) [inline nflgame/version.py]: Looks like your `__version__` may have reverted to an older numbering.

File: nflgame/update_players.py [hunk 3]
```diff
@@ -455,13 +455,24 @@ def fetch(purl):
 
         def fetch(t):
             gid, purl = t
-            resp, content = new_http().request(purl, 'GET')
-            if resp['status'] != '200':
-                if resp['status'] == '404':
-                    return gid, purl, False
+            resp = requests.get(purl)
+            if resp.status_code != 200:
+                if resp.status_code != 404:
```
Comment (mickeelm) [inline nflgame/update_players.py]: Old code has ==, you've put in !=

File: nflgame/update_players.py (hunk 1 above)
Comment (NickBenthem) [reply nflgame/update_players.py]: Should be fixed.

File: nflgame/version.py (hunk 2 above)
Comment (NickBenthem) [reply nflgame/version.py]: Huh. Weird.

Fixed to be higher.

File: nflgame/update_players.py (hunk 3 above)
Comment (NickBenthem) [reply nflgame/update_players.py]: Should be fixed now :)

**Comment by derek-adair** [comment]
I think this has been resolved. There were a bunch of PRs for this and other bugs.   I'll be comparing and making sure this is fixed.

**Pull request closed without merging**